pkg/registry/apps/statefulset/strategy_test.go

```diff
@@ -21,6 +21,8 @@ import (
 
 	"github.com/google/go-cmp/cmp"
 
+	"k8s.io/utils/ptr"
```

Comment:
The import has been moved. Guess goimport sorted this, but we should put it back where it has been before 👍